docs/ENVIRONMENT.rst

```diff
@@ -0,0 +1,54 @@
+==================================
+Environment Configuration Settings
+==================================
+
+Some of configuration parameters defined in the configuration file is possible to override via Environment variables
+This document list all possible environment variables handled by Patroni.
+Environment variable always takes precedence on configuration file.
+
+Global/Universal
+----------------
+-  **PATRONI\_NAME**: name of the node where the current instance of Patroni is running. Must be unique for the cluster.
+-  **PATRONI\_NAMESPACE**: path within configuration store where Patroni will keep information about cluster. Default value: "/service"
+-  **PATRONI\_SCOPE**: cluster name
+
+Bootstrap configuration
+-----------------------
+It is possible to define users which will be created right after initializing of a new cluster by defining following environment variables:
```

Comment:
I'd change the following to:

It is possible to create new database users right after the successful initialization of a new cluster. This process is defined by the following variables: